pkg/compiler/check.go

```diff
@@ -34,6 +35,21 @@ func (comp *compiler) check() {
 	comp.checkDupConsts()
 }
 
+func (comp *compiler) checkComments() {
+	prefixes := []string{"include", "define", "incdir"}
+	for _, decl := range comp.desc.Nodes {
+		switch n := decl.(type) {
+		case *ast.Comment:
+			text := strings.TrimSpace(n.Text)
+			for _, prefix := range prefixes {
+				if strings.HasPrefix(text, prefix) {
+					comp.error(n.Pos, "confusing comment faking a directive (rephrase if it's intentional)")
```

Comment:
Thank you! hopefully, it'd save someone in the future 👍

Reply:
It will definitely save me in future :)